examples/form/src/App.js

```diff
@@ -25,23 +26,31 @@ class App extends Component {
         min: 0.25,
         max: 0.75
       },
-      steps: new Array(21).fill().map((e,i) => i * 0.05)
+      steps: new Array(21).fill().map((e,i) => i * 0.05),
+      textbox: ''
     };
     this.clickButton = this.clickButton.bind(this);
     this.onChangeCheckbox1 = this.onChangeCheckbox1.bind(this);
     this.onChangeCheckbox2 = this.onChangeCheckbox2.bind(this);
+    this.onChangeTextbox = this.onChangeTextbox.bind(this);
     this.onChangeSinglePicker = this.onChangeSinglePicker.bind(this);
     this.onChangeMultiplePicker = this.onChangeMultiplePicker.bind(this);
     this.onMouseLeaveRow = this.onMouseLeaveRow.bind(this);
     this.onMouseEnterRow = this.onMouseEnterRow.bind(this);
     this.setSliderHandleChange = this.setSliderHandleChange.bind(this);
   }
 
-  onChangeCheckbox1() {
+  // The functions that are called when change directlly
+  // the textbox or checkbox recive an event as param
+  onChangeTextbox(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  onChangeCheckbox1(event) {
     this.setState({ checked: !this.state.checked });
   }
 
-  onChangeCheckbox2() {
+  onChangeCheckbox2(event) {
```

Comment:
argument event is not being used

Reply:
i know, for that reason i put a comment that describe why the function receive a param, i think the people can think that GUIM lost the react events when change an input

Reply:
maybe add a underscore prefix? a la elixir
`onChangeCheckbox2(_event)`

Reply:
i like the idea